### DISCUSSION TITLE
Why is tsx file icon not shown

### DISCUSSION THREAD

**Discussion by tokisakiyuu** [LazyVim/LazyVim#4095] [Q&A] [answered]
<img width="1246" alt="截屏2024-07-18 09 56 47" src="https://github.com/user-attachments/assets/947c33de-31c4-4983-822f-9e226a7f4a98">

### Version
```bash
NVIM v0.10.0
Build type: Release
LuaJIT 2.1.1713773202
```

**Comment by folke** [comment, marked as answer]
You probably are not using a Nerd Font v3. This is how it should look like:

![image](https://github.com/user-attachments/assets/cdc3ca03-5882-419a-995e-fa01597e31b1)

**Reply by tokisakiyuu** [reply]
Thanks! I just installed `font-hack-nerd-font` using brew and the problem was fixed!
